superset-frontend/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -165,13 +165,25 @@ class PropertiesModal extends React.PureComponent {
   }
 
   async handleErrorResponse(response) {
-    const { error, statusText } = await getClientErrorObject(response);
+    const { error, statusText, message } = await getClientErrorObject(response);
+    let errorText = error || statusText || t('An error has occurred');
```

Comment:
Could be good to refactor that message and similar ones in a constant:
```bash
$ git grep -i "error.*occurred'" superset-frontend/
superset-frontend/src/SqlLab/components/ExploreCtasResultsButton.jsx:          this.props.errorMessage || t('An error occurred'),
superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx:          this.props.errorMessage || t('An error occurred'),
superset-frontend/src/components/TableLoader.tsx:        props.addDangerToast(t('An error occurred'));
superset-frontend/src/dashboard/components/PropertiesModal.jsx:      body: error || statusText || t('An error has occurred'),
superset-frontend/src/datasource/DatasourceEditor.jsx:            error || statusText || t('An error has occurred'),
superset-frontend/src/datasource/DatasourceModal.tsx:            body: error || t('An error has occurred'),
superset-frontend/src/explore/components/DisplayQueryButton.jsx:            error: error || statusText || t('Sorry, An error occurred'),
superset-frontend/src/explore/components/PropertiesModal.tsx:      body: error || statusText || t('An error has occurred'),
superset-frontend/src/utils/getClientErrorObject.ts:            : t('An error occurred');
```

Reply:
Agreed, let's treat that as a separate task.